src/wallet/wallet.h

```diff
@@ -676,23 +676,18 @@ class COutput
     }
 };
 
-/** Private key that includes an expiration date in case it never gets used. */
-class CWalletKey
-{
-public:
+/** Private key that was serialized by an old wallet (only used for deserialization) */
+struct OldKey {
     CPrivKey vchPrivKey;
-    int64_t nTimeCreated;
-    int64_t nTimeExpires;
-    std::string strComment;
-    // todo: add something to note what created it (user, getnewaddress, change)
-    //   maybe should have a map<string, string> property map
-
-    explicit CWalletKey(int64_t nExpires=0);
-
     ADD_SERIALIZE_METHODS;
 
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action) {
```

Comment:
fa6f22bf44c0f741285f27f27ac18e9679802e5e

Could go further and just implement `Unserialize`?

Reply:
I kept the changes to a minimum. Happy to add a commit for this, if other think it is useful.